django/contrib/postgres/search.py

```diff
@@ -365,6 +365,16 @@ class TrigramWordDistance(TrigramWordBase):
     function = ""
     arg_joiner = " <<-> "
 
+    def __int__(self, string, expression, strict=False, **extra):
+        if strict:
+            self.arg_joiner = " <<<-> "
+        super().__init__(string, expression, **extra)
+
 
 class TrigramWordSimilarity(TrigramWordBase):
     function = "WORD_SIMILARITY"
+
+    def __int__(self, string, expression, strict=False, **extra):
+        if strict:
+            self.function = "STRICT_WORD_SIMILARITY"
+        super().__init__(string, expression, **extra)
```

Comment:
The only thing that we change  is a `function`/`arg_joiner` so I would add a separate classes:
```python
class TrigramStrictWordDistance(TrigramWordBase):
    function = ""
    arg_joiner = " <<<-> "


class TrigramStrictWordSimilarity(TrigramWordBase):
    function = "STRICT_WORD_SIMILARITY"
```